src/mlpack/methods/pca/pca_impl.hpp

```diff
@@ -74,6 +75,20 @@ void PCA<DecompositionPolicy>::Apply(const arma::mat& data,
   arma::mat eigvec;
   Apply(data, transformedData, eigVal, eigvec);
 }
+ /*This is another Overload of apply with only 2 parameteres(data & transformed data)
+ and it will create eigval and eigvec and store the corresponding values  in them
+ as the source of information are first 2 parameters only.
+ * @param data - Data matrix
+ * @param transformedData - Data with PCA applied
+ */
```

Comment:
Can you take a look at the style of the other blocks, and apply it here.

Reply:
@zoq I have done the style changes as suggested by you.